x-pack/test/functional_basic/apps/transform/group2/config.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FtrConfigProviderContext } from '@kbn/test';
+
+export default async function ({ readConfigFile }: FtrConfigProviderContext) {
+  const xpackFunctionalConfig = await readConfigFile(
+    require.resolve('../../../../functional/config.base.js')
+  );
+
+  return {
+    // default to the xpack functional config
+    ...xpackFunctionalConfig.getAll(),
+    esTestCluster: {
+      ...xpackFunctionalConfig.get('esTestCluster'),
+      license: 'basic',
+      serverArgs: [
+        'xpack.license.self_generated.type=basic',
+        'xpack.security.enabled=true',
+        'xpack.security.authc.api_key.enabled=true',
+      ],
+    },
+    testFiles: [require.resolve('.')],
+    junit: {
+      ...xpackFunctionalConfig.get('junit'),
+      reportName: 'Chrome X-Pack UI Functional Tests Basic License - transform',
+    },
```

Comment:
So, there is a considerable amount of config copies between all of these config files. In the future I would probably create a `transform/config.base.ts` files that all of these could extend to avoid copying settings between all of these files.

Reply:
Totally agree, I will fix that.